.github/workflows/deploy.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and Publish Storybook to GitHub Pages
+
+on:
+  push:
+    branches:
+      - develop
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+env: 
+  STORYBOOK_RC_HOST: "https://demo.qa.rocket.chat"
+  
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "16.19.0"
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Build Storybook
+        run: yarn build:storybook
+        working-directory: packages/react
+
+      - name: Build UI-Elements
+        run: yarn build:storybook
+        working-directory: packages/ui-elements
+
+      - name: Build Layout Editor
+        run: npm run build
+        working-directory: packages/layout_editor
+
+      - name: Setup Node.js for Docs
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18.x"
+
+      - name: "Install dependencies for docs"
+        run: yarn install
+        working-directory: packages/docs/
+
+      - name: Build Docs
+        run: yarn run build
```

Comment:
Why do we need to run `yarn run build`, I think `yarn build` will also work ?

Reply:
done